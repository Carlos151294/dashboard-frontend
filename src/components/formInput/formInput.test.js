import React from 'react';
import { shallow } from 'enzyme';
import FormInput from './formInput.component';

it('Excepect to render FormInput component with value, label and hint', () => {
    const mockLabel = 'My label';
    const mockValue = 'My value';
    const mockHint = 'My hint';
    expect(shallow(<FormInput value={mockValue} label={mockLabel} hint={mockHint} />)).toMatchSnapshot();
});

it('Excepect to render FormInput component with no hint', () => {
    const mockLabel = 'My Label';
    const mockValue = 'My value';
    expect(shallow(<FormInput value={mockValue} label={mockLabel}/>)).toMatchSnapshot();
});

it('Excepect to render FormInput component with only hint', () => {
    const mockHint = 'My hint';
    expect(shallow(<FormInput hint={mockHint} />)).toMatchSnapshot();
});

it('Excepect to render FormInput component with no properties', () => {
    expect(shallow(<FormInput />)).toMatchSnapshot();
});