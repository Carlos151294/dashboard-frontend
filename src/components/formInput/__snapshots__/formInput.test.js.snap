// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Excepect to render FormInput component 1`] = `ShallowWrapper {}`;

exports[`Excepect to render FormInput component with both label and hint 1`] = `ShallowWrapper {}`;

exports[`Excepect to render FormInput component with label and hint 1`] = `ShallowWrapper {}`;

exports[`Excepect to render FormInput component with no hint 1`] = `ShallowWrapper {}`;

exports[`Excepect to render FormInput component with no properties 1`] = `ShallowWrapper {}`;

exports[`Excepect to render FormInput component with only hint 1`] = `ShallowWrapper {}`;

exports[`Excepect to render FormInput component with only label 1`] = `ShallowWrapper {}`;

exports[`Excepect to render FormInput component with value, label and hint 1`] = `ShallowWrapper {}`;
